### ISSUE TITLE
Possibility to specify additional category instead of exclusive category

### ISSUE THREAD

**Issue by mvgijssel** [Cube777/dotgit#2]
First of all, thanks for this awesome tool! I've tried almost all the dotfile managers out there and this one just works beautifully! 

I was wondering, as a nice to have feature, would it be possible to define an additional category? Currently specifying a category ignores everything else, but I'd like it to also support "also symlink these files".

The use case for me is work related, being able to store my work related configuration in a separate directory. Something like:
```
HOSTNAME=macbook dotgit update +work
=> symlink dotfiles/common
=> symlink dotfiles/macbook
=> symlink dotfiles/work
```
versus (current behaviour)
```
HOSTNAME=macbook dotgit update work
=> symlink dotfiles/work
```

Let me know if this is possible or if I'm missing something!

**Comment by Cube777** [comment]
Great to hear you enjoy using dotgit 👍 I too have thought about implementing this and I don't think it should be to hard but I'm currently a bit busy (getting ready for university). As soon as I get some time I'll implement this.

**Comment by Cube777** [comment]
Hey @mvgijssel , I finally got around to rewriting dotgit and I'm now glad to say it supports multiple categories :+1: Please have a look at [this post](https://bbs.archlinux.org/viewtopic.php?pid=1720722#p1720722) for instructions. Have a good day!

**Comment by Cube777** [comment]
Fixed in 1.4.1
